tests/test_fxa_test_account.py

```diff
@@ -0,0 +1,46 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+import re
+
+import pytest
+from fxa.errors import ClientError
+
+from fxapom.fxapom import DEV_URL, FxATestAccount, PROD_URL
+
+
+@pytest.mark.skip_selenium
+class TestFxATestAccount(object):
+
+    def check_login(self, acct):
+        # attempt to login, if unsuccessful will throw a ClientError
+        try:
+            acct.login()
+        except ClientError:
+            raise AssertionError('Login unsuccessful. Account not properly created/verified.')
+
+    def test_default_environment_should_be_dev(self):
+        acct = FxATestAccount()
+        assert DEV_URL == acct.url
+
+    def test_create_new_account_on_dev(self, dev_account):
+        assert dev_account.is_verified
+        assert DEV_URL == dev_account.url
+        self.check_login(dev_account)
+
+    def test_create_new_account_on_prod(self, prod_account):
+        assert prod_account.is_verified
+        assert PROD_URL == prod_account.url
+        self.check_login(prod_account)
+
+    def test_new_account_pw_does_not_contain_numbers(self, dev_account):
+        test_regex = re.compile('\d')
+        assert test_regex.search(dev_account.password) is None
```

Comment:
I don't think we really need to compile the regex here, we can just use [re.search](https://docs.python.org/2/library/re.html#re.search)